docs/pages/store/reference/misc.mdx

```diff
@@ -269,6 +269,62 @@ function slice32(bytes memory data, uint256 start) internal pure returns (bytes3
 | -------- | --------- | -------------------------------------------------------------------------- |
 | `output` | `bytes32` | The extracted bytes32 value from the specified position in the bytes blob. |
 
+## FieldLayout_Empty
```

Comment:
I feel like I saw this in my PR too and IIRC had to clear out all the various foundry caches (cache, out, docs) before rebuilding/regenerating docs. Try that and see if this disappears?

Reply:
Good idea, I've been playing with it to no results

Reply:
also be sure this has latest main too